ui/app/AppLayouts/Chat/components/ProfilePopup.qml

```diff
@@ -143,6 +143,15 @@ ModalPopup {
             anchors.topMargin: Theme.smallPadding
         }
 
+        CopyToClipBoardIcon {
+          anchors.top: labelEnsUsername.bottom
+          anchors.left: valueEnsName.right
+          anchors.leftMargin: Theme.smallPadding
+          onClick: function (){
+            chatsModel.copyToClipboard("@emily.stateofus.eth")
```

Comment:
Can this be replaced by `chatsModel.copyToClipboard(valueEnsName.text)` ?

Reply:
Oh yes sure. Just put it as a placeholder for now, but lemme update this